tests/ut/test_decorators.py

```diff
@@ -94,6 +94,37 @@ def test_get_cache_key_with_key_builder(self, decorator):
         assert decorator.cache.set.call_count == 0
         assert stub.call_count == 0
 
+    @pytest.mark.asyncio
+    async def test_disable_read(self, decorator, decorator_call):
+        decorator.cache.get = CoroutineMock(return_value=1)
```

Comment:
Why do you need this if its not being called?

Reply:
Probably from copy&pasting, removed